audit/src/test/resources/audit.yml

```diff
@@ -0,0 +1,61 @@
+# AuditHandler will pick some important fields from headers and tokens and logs into a audit appender.
+---
+# Enable Audit
+enabled: true
+
+# Enable mask
+mask: false
+
+# Output response status code
+statusCode: true
+
+# Output response time
+responseTime: true
+
+# audit only on error responses; by default log for any return status code
+# if true, audit at error level if status code >= 400
+# log level is controlled by logLevel
+auditOnError: false
+
+# log level; by default set to info
+logLevelIsError: false
+
+
+# Output header elements. You can add more if you want.
+headers:
+
+# Correlation Id
+- X-Correlation-Id
+
+# Traceability Id
+- X-Traceability-Id
+
+# Output from id token and access token
+audit:
+
+# Client Id
+- client_id
+
+# User Id in id token, this is optional
+- user_id
+
+# Client Id in scope/access token, this is optional
+- scope_client_id
+
+# Request endpoint uri@method.
+- endpoint
+
+# Service ID assigned to the service, this is optional and must be set by the service in its implementation
+- serviceId
+
+# Request Body, this is optional and must be set by the service in its implementation
+- request
```

Comment:
can we rename this to requestBody and responseBody? Just saying request  sounds like it will audit the whole request including query params, path params, cookies etc

Reply:
Thanks for pointing out, I also think renaming would be better. But seems these names have been used in previous versions. If we change them now, it may cause backward compatibility. People who previously used these names to audit requests may not be able to continue auditing them if they don't change their configuration. @stevehu Do you think we can change it?

Reply:
I think it make sense to rename it to clearly define what is the audited object. Regarding the backward compatibility in the config, we can mention that in the next release notes. It is not a coding issue but only the configuration file and it is easier for users to adjust. Thank you for pointing it out.

Reply:
I see. Will rename them.